src/settings.cpp

```diff
@@ -393,5 +399,38 @@ void Settings::setNmeaSerialPort(const QString& name)
 	}
 }
 
+std::vector<QColor> Settings::colorsStringToVector(QString config_string)
+{
+	auto const color_strings = config_string.split(QLatin1Char(','));
```

Comment:
`...splitRef(...`

Reply:
Fixed. I keep forgetting about this one.